src/Features/LanguageServer/Protocol/LspOptionsStorage.cs

```diff
@@ -20,5 +20,13 @@ internal sealed class LspOptionsStorage
 
         // Flag is defined in VisualStudio\Core\Def\PackageRegistration.pkgdef.
         public static readonly Option2<bool> LspSemanticTokensFeatureFlag = new("dotnet_enable_lsp_semantic_tokens", defaultValue: false);
+
+        /// <summary>
+        /// This flag is turned on when the C# devkit is installed.
+        /// This can cause certain LSP features to behave differently, for example we avoid returning test code lenses when devkit is running.
+        /// 
+        /// This flag is not user visible.
+        /// </summary>
+        public static readonly Option2<bool> LspUsingDevkitFeatures = new("dotnet_lsp_using_devkit", defaultValue: true);
```

Comment:
Since DevKit is an optional extension, we should default to `false`.

```suggestion
        public static readonly Option2<bool> LspUsingDevkitFeatures = new("dotnet_lsp_using_devkit", defaultValue: false);
```